turbinia/api/routes/request.py

```diff
@@ -16,23 +16,25 @@
 
 import logging
 import uuid
+import json
 
 from fastapi import HTTPException, APIRouter
 from fastapi.responses import JSONResponse
-
+from fastapi.requests import Request
 from pydantic import ValidationError
 from turbinia import TurbiniaException, client as turbinia_client
 from turbinia import evidence
 from turbinia.lib import recipe_helpers
-from turbinia.api.schemas import request
+from turbinia.api.schemas import request as turbinia_request
 from turbinia.api.models import request_status
 
-log = logging.getLogger('turbinia:api_server:models:request')
+log = logging.getLogger('turbinia')
+
 router = APIRouter(prefix="/request", tags=["Turbinia Requests"])
 
 
-@router.get("/summary", response_model=request_status.RequestsSummary)
-async def get_requests_summary():
+@router.get("/summary")
+async def get_requests_summary(request: Request):
```

Comment:
Not presently, but the FastAPI framework makes it available by default due to the use of the decorator. I figured it would be good to be explicit about it existing.